src/internal/defaultMiddlewares.ts

```diff
@@ -0,0 +1,13 @@
+import { CallServiceArg, CallServiceData, CallServiceResult, Middleware, ResultCodes } from './CallServiceHandler';
+
+/**
+ * Error catching middleware
+ */
+export const errorHandler: Middleware = (req: CallServiceData, resp: CallServiceResult, next: Function): void => {
+    try {
+        next();
+    } catch (e) {
+        resp.retCode = ResultCodes.exceptionInHandler;
+        resp.result = e.toString();
```

Comment:
Let's add more debug info to the error? Particle id maybe? service id and fnName at least.

Reply:
ok